### ISSUE TITLE
Multiple ESP8266's, one alexa device

### ISSUE THREAD

**Issue by ThatCarrot** [Aircoookie/Espalexa#219]
I think the title is pretty self explenatory, but i'll clarify here

My question is: given two esp8266's, each running an instance of an alexa device, will there be conflicts?
what i want to achieve is saying "alexa turn on LEDs in my room" to light on two different strips of LEDs in my room.

thank you in advcance, also sorry for any grammatical errors.
Lorenzo

**Comment by BoeckEH** [comment]
So, probably not a good idea. Instead, I'd make a routine that turns them both on separately. This also leaves room to only do one at a time in the future without any programming changes, if you change your mind. Or add another strip easily. Just my $0.02.

**Comment by MakerMatt46307** [comment]
Agreed. Cheap shortcut, get two smart Kasa plugs and use their Kasa app or
Alexa to run a routine.
